lib/session/session_test.go

```diff
@@ -40,52 +52,47 @@ type SessionSuite struct {
 	clock clockwork.FakeClock
 }
 
-var _ = Suite(&SessionSuite{})
-
-func (s *SessionSuite) SetUpSuite(c *C) {
-	utils.InitLoggerForTests(testing.Verbose())
-}
-
-func (s *SessionSuite) SetUpTest(c *C) {
+func NewSessionSuite(t *testing.T) *SessionSuite {
 	var err error
+	s := &SessionSuite{}
 
 	s.clock = clockwork.NewFakeClockAt(time.Date(2016, 9, 8, 7, 6, 5, 0, time.UTC))
-	s.dir = c.MkDir()
-
+	s.dir = t.TempDir()
 	s.bk, err = lite.NewWithConfig(context.TODO(),
 		lite.Config{
 			Path:  s.dir,
 			Clock: s.clock,
 		},
 	)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 
 	srv, err := New(s.bk)
+	require.NoError(t, err)
 	srv.(*server).clock = s.clock
 	s.srv = srv.(*server)
-	c.Assert(err, IsNil)
+	return s
 }
 
-func (s *SessionSuite) TearDownTest(c *C) {
-	c.Assert(s.bk.Close(), IsNil)
+func TearDownSessionSuite(s *SessionSuite, t *testing.T) {
```

Comment:
This could be a method on the suite type.

Reply:
Adjusted